src/plays/inspirational-quotes/card.css

```diff
@@ -0,0 +1,72 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,400&display=swap");
+*,
```

Comment:
No * level styles at the play level. It will override things at the base. Scope all the styles in this file with a unique class name prefix.

Reply:
got it.